x-pack/plugins/siem/public/components/timeline/data_providers/provider_badge.tsx

```diff
@@ -88,8 +88,15 @@ export const ProviderBadge = pure<ProviderBadgeProps>(
         {prefix}
         {operator !== EXISTS_OPERATOR ? (
           <>
-            <span className="field-value">{field}: </span>
-            <span className="field-value">&quot;{val}&quot;</span>
+            <span className="field-value">
+              {field}
+              {': '}
+            </span>
+            <span className="field-value">
+              {'"'}
+              {val}
+              {'"'}
+            </span>
```

Comment:
This testing is on the providers:
<img width="279" alt="Screen Shot 2019-05-30 at 9 44 22 AM" src="https://user-images.githubusercontent.com/1151048/58645022-bc3d2900-82bf-11e9-8545-7dd96bd36469.png">

I should be able to use template interpolation to make this not as gross:

```ts
<span className="field-value">{`"${val}"`}</span>
```